src/main/scala/org/broadinstitute/dsde/rawls/webservice/UserApiService.scala

```diff
@@ -92,6 +92,13 @@ trait UserApiService extends HttpService with PerRequestCreator with UserInfoDir
           UserService.props(userServiceConstructor, userInfo),
           UserService.AdminDisableUser(RawlsUserRef(RawlsUserSubjectId(userSubjectId))))
       }
+    } ~
+    path("user" / Segment / "remove") { userSubjectId =>
```

Comment:
Yep, DELETE makes sense here, more so than disable.  That keeps some info about the user in the system.